test/Maestro/EU/NL_UserRegistration_Successful.yml

```
@@ -1,40 +1,59 @@
     label: "Enter City/Town"
-    text: "Lelystad"
+    text: ${CITY}
 - tapOn: "Mobile number"
 - inputText:
     label: "Enter Phonenumber"
-    text: "612345678"
-- tapOn: "IBAN account number\nIBAN account number"
-- inputText:
+    text: ${PHONENUMBER}
+- tapOn: 
     label: "Enter IBAN"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)